acquire_zarr/CMakeLists.txt

```
@@ -45,6 +49,9 @@ install(TARGETS zarr_writer_node
         RUNTIME DESTINATION bin
 )
 
+# Install the external acquire-zarr library
+install(FILES ${ACQUIRE_ZARR_INSTALL_DIR}/lib/libacquire-zarr${CMAKE_SHARED_LIBRARY_SUFFIX} DESTINATION lib)
```

I think you can use `${CMAKE_SHARED_LIBRARY_PREFIX}` here if you're targeting windows too.
```suggestion
install(FILES ${ACQUIRE_ZARR_INSTALL_DIR}/lib/${CMAKE_SHARED_LIBRARY_PREFIX}acquire-zarr${CMAKE_SHARED_LIBRARY_SUFFIX} DESTINATION lib)
```